doc/developers/advanced_installation.rst

```diff
@@ -87,7 +87,7 @@ feature, code or documentation improvement).
 
      python3 -m venv sklearn-env
      source sklearn-env/bin/activate
-     pip install wheel numpy scipy cython
+     pip install wheel numpy scipy cython meson-python
```

Comment:
Is ninja also installed automatically with pip? It wasn't the case in the past but I haven't tried again recently.